docs/source/java/quickstartguide.rst

```diff
@@ -0,0 +1,325 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+.. default-domain:: java
+.. highlight:: java
+
+=================
+Quick Start Guide
+=================
+
+.. contents::
+
+Arrow Java provides several building blocks. Data types describe the types of values;
+``ValueVectors`` are sequences of typed values; ``fields`` describe the types of columns in
```

Comment:
We don't need to put any of these terms in a code font

Reply:
Changed